app/src/ui/history/commit-list.tsx

```diff
@@ -189,9 +189,11 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
         key={commit.sha}
         gitHubRepository={this.props.gitHubRepository}
         isLocal={isLocal}
-        canBeUndone={this.props.canUndoCommits && isLocal && row === 0}
-        canBeAmended={this.props.canAmendCommits && row === 0}
-        canBeResetTo={this.props.canResetToCommits && isResettableCommit}
+        canBeUndone={this.props.canUndoCommits === true && isLocal && row === 0}
+        canBeAmended={this.props.canAmendCommits === true && row === 0}
+        canBeResetTo={
+          this.props.canResetToCommits === true && isResettableCommit
+        }
```

Comment:
Yes. Not for booleans if we require the boolean (thus it can only be true or false). But those props became boolean | undefined. I think I was requested before to make it === true so that it is clear that you must say true for this to be true => thus false or undefined both definitely mean false.